src/raftstore/store/store.rs

```diff
@@ -1062,15 +1063,15 @@ impl<T: Transport, C: PdClient> Store<T, C> {
 
         let res = peer.check_epoch(msg);
         if let Err(Error::StaleEpoch(msg, mut new_regions)) = res {
-            // Attach the next region which might be split from the current region. But it doesn't
-            // matter if the next region is not split from the current region. If the region meta
-            // received by the TiKV driver is newer than the meta cached in the driver, the meta is
-            // updated.
-            if let Some((_, &next_region_id)) = self.region_ranges
-                .range(Excluded(&enc_end_key(peer.region())), Unbounded::<&Key>)
-                .next() {
-                let next_region = self.region_peers[&next_region_id].region();
-                new_regions.push(next_region.to_owned());
+            // Attach the previous region which might be split from the current region. But it
+            // doesn't matter if the previous region is not split from the current region. If the
+            // region meta received by the TiKV driver is newer than the meta cached in the
+            // driver, the meta is updated.
```

Comment:
`the meta is updated` means `the cache is updated`? where does this update happen?